src/modules/client.ts

```diff
@@ -147,7 +147,7 @@ export default class Client {
 
   executeMutation$(
     mutationObject: IQuery
-  ): Observable<IExchangeResult['data']> {
+  ): Observable<IExchangeResult> {
```

Comment:
This will also need to change in more locations. There should be a client interface and the client wrapper might also refer to it a couple of times